graviti/manager/branch.py

```diff
@@ -61,31 +59,32 @@ class BranchManager:
     """This class defines the operations on the branch in Graviti.
 
     Arguments:
-        dataset: :class:`~graviti.dataset.dataset.Dataset` instance.
+        dataset: :class:`~graviti.manager.dataset.Dataset` instance.
 
     """
 
     def __init__(self, dataset: "Dataset") -> None:
         self._dataset = dataset
-
-    def _generate(
-        self, name: Optional[str] = None, offset: int = 0, limit: int = 128
-    ) -> Generator[TensorbayBranch, None, int]:
+        self._commonly_used_params = {
```

Comment:
why group the params into a dict?

Reply:
These parameters are used in almost every function. Putting them into a dict reduces code duplication.